CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## v1.0.0-alpha.2
+
+### Additions
+
+- The Ansible and Helm operators have a `version` subcommand that prints the version information for the `ansible-operator` and `helm-operator` binaries. ([#3596](https://github.com/operator-framework/operator-sdk/pull/3596))
+- Added "--ansible-args" command-line flag that allows users to specify arbitrary CLI arguments for ansible-based operators that are passed through ansible-runner. For example, passing --ansible-vault as an arbitrary argument allows user to store sensitive data in encrypted files. ([#3374](https://github.com/operator-framework/operator-sdk/pull/3374))
```

Comment:
Replace the double quotes around --ansible-args with backticks `

ansible-based should be capitalized:Ansible-based

--ansible-vault should be wrapped in backticks